api/src/main/java/com/community/api/endpoint/avisoft/controller/Admin/RoleController.java

```diff
@@ -0,0 +1,41 @@
+package com.community.api.endpoint.avisoft.controller.Admin;
+
+import com.community.api.entity.Role;
+import com.community.api.services.RoleService;
+import com.community.api.services.exception.ExceptionHandlingImplement;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.ParameterResolutionDelegate;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping(value = "/roles",
+        produces = { MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_XML_VALUE }
+)
+public class RoleController {
+    @Autowired
+    private RoleService roleService;
+    @Autowired
+    private ExceptionHandlingImplement exceptionHandling;
+    @PostMapping("addRole")
+    public ResponseEntity<?> getRoleList(@RequestBody Role role)
```

Comment:
update your action addRole => to createRole in postmapping and function name both